cloud_blobstore/gs.py

```diff
@@ -6,7 +6,53 @@
 from google.cloud.storage import Client
 from google.cloud.storage.bucket import Bucket
 
-from . import BlobNotFoundError, BlobStore
+from . import BlobNotFoundError, BlobStore, PagedIter
+
+
+class GSPagedIter(PagedIter):
+    def __init__(
+            self,
+            bucket_obj, # type: Bucket
+            prefix: str=None,
```

Comment:
Since this has a ton of arguments, I would suggest using a `*` argument to require that subsequent arguments be keyword arguments.